icubam/backoffice/translations/es.csv

```diff
@@ -0,0 +1,101 @@
+Active,Activo
+Admin,Administrador ICUBAM
+Authentication failed,Error de autenticación
+Create user,Nuevo usuario
+Edit user,Edición de usuario
+Email address,Email
+Failed to save the user,Error de registro de usuario
+Select ICUs,Escoger UCIs
+Name,Nombre
+Password,Contraseña
+Phone Number,Número telefónico
+Save,Guardar
+Sign in to start your session,Identificarse e ingresar
+Sign In,Ingresar
+User management,Manejo de usuarios
+n_covid_occ,Covid+ ocupadas
+n_ncovid_occ,Covid- ocupadas
+n_covid_free,Covid+ libres
+n_ncovid_free,Covid- libres
+n_covid_deaths,Muertes
+n_covid_healed,Sanados/Curados
+n_covid_refused,Rechazados
+n_covid_transfered,Transferidos
+
+Bed Counts,Recuento de camas
+Users,Usuarios
+ICUs,UCIs
+Dashboards,Tableros
+Map,Mapa
+Operational dashboard,Tablero operacional
+Access Tokens,Tokens de accesos
+Regions,Regiones
+Messages,Mensajes
+
+Manage ICUs,Gestionar UCIs
+Edit,Editar
+Create,Crear
+Manage ICU,Gestionar UCI
+
+Home,Inicio
+Logout,Desconectarse
+All rights reserved,Todos los derechos reservados
+Where is the ICU?,¿Dónde está la UCI?
+
+Overall,Resumen
+icus,UCIs
+users,Usuarios
+Occupied,Ocupadas
+Free,Disponibles
+deceased,Fallecidos
+Healed,Recuperados
+
+icu (update link),UCI (Link de actualización)
+since_update,Última actualización
+
+Add,Crear
+dataTables.english.lang,//cdn.datatables.net/plug-ins/1.10.20/i18n/Spanish.json
+
+name,Nombre
+city,Ciudad
+dept,Comuna
+region,Región
+active,Activa
+managers,Administradores
+
+admin,Administrador ICUBAM
+created,Creación
+manages,Administra
+
+Actif,Activo
+Managed ICUs,UCIs administradas
+Already set,Ya establecida
+Cancel,Cancelar
+
+Legal ID,RUT
+Region,Región
+City,Ciudad
+Department / State,Comuna
+Latitude,Latitud
+Longitude,Longitud
+Submit,Enviar
+
+Manage Regions,Gestionar regiones
+
+Clicker pour afficher,Haga click para mostrar todo
+Tout montrer,Mostrar todo
+
+Occupied COVID+ beds,Camas COVID+ ocupadas
+Available COVID+ beds,Camas COVID+ disponibles
+COVID deceased,Fallecidos COVID
+COVID healed,Recuperados COVID
+Refused,Rechazados
+(due to unavailability),(por capacidad)
+Transfers,Transferidos
+(to other ICU),(a otras UCIs)
+Toutes les régions,Todas las regiones
+
+Taux de refus (covid) / capacité totale,Tasa de rechazo (COVID) / capacidad total
+Taux de remplissage lits (total),Tasa de ocupación de camas (total)
+Taux de remplissage lits (covid),Tasa de ocupación de camas (COVID)
+Fraction (%),Fracción (%)
```

Comment:
awesome !

Reply:
If there are french strings in the code, it would be better to translate them to english and have the translation file be en->es, and we will update the en->fr file, no?

Reply:
Yeah, for the sake of consistency I agree. I'll change that.